packages/e2e_tests/cypress/e2e/step_definitions/Register-to-have-your-say/user-registers-to-have-their-say.cy.js

```diff
@@ -0,0 +1,74 @@
+import { PO_HaveYourSay } from '../../../pageObject/Register-to-have-your-say/PO_HaveYourSay';
+const haveYourSay = new PO_HaveYourSay();
+
+describe('User registers to have their say and submits a comment against a project', () => {
+	it('Navigate to projects search page', () => {
+		cy.visit('https://applications-service-test.planninginspectorate.gov.uk/project-search/');
```

Comment:
Suggestion: Set the baseUrl in `cypress.config.js` so you only have to add the path to the visit function i.e `cy.visit('/project-search')`